projects/js-packages/charts/src/components/bar-chart/bar-chart.tsx

```diff
@@ -102,7 +118,12 @@ const BarChart: FC< BarChartProps > = ( {
 	} ) );
 
 	return (
-		<div className={ clsx( 'bar-chart', className, styles[ 'bar-chart' ] ) }>
```

Comment:
We added a string class to make the debugging easier for implementers and allow external CSS changes if needed.

Reply:
Ah, thank you! 🙏